### PR TITLE
Integrate santuario 2.1.7 and its dependencies (#83)

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: wsit/boms/bom-ext/pom.xml [hunk 1]
```diff
@@ -32,12 +32,11 @@
         <connector-api.version>1.7.4</connector-api.version>
         <ejb-api.version>3.2.6</ejb-api.version>
         <transaction-api.version>1.3.3</transaction-api.version>
-        <jaxws.home.uri>http://eclipse-ee4j.github.io/metro-jax-ws</jaxws.home.uri>
```
Comment (lukasj) [inline wsit/boms/bom-ext/pom.xml:35]: this must stay, the property is used in the docs and replaced by filtering...

Comment (Tomas-Kraus) [reply wsit/boms/bom-ext/pom.xml:35]: Check the whole file. Last property in the list is
`<jaxws.home.uri>http://eclipse-ee4j.github.io/metro-jax-ws</jaxws.home.uri>`
I just removed the redundant one.

**Review by lukasj — APPROVED** [review]
LGTM

File: wsit/boms/bom-ext/pom.xml (hunk 1 above)
Comment (lukasj) [reply wsit/boms/bom-ext/pom.xml:35]: right

**Pull request merged**